src/encoded/static/components/static-pages/HomePage.js

```diff
@@ -28,23 +29,28 @@ export default class HomePage extends React.PureComponent {
         "session": PropTypes.bool.isRequired
     };
 
-    introText(){
-        var introContent = _.findWhere(this.props.context.content, { 'name' : 'home.introduction' }); // Content
-
-        if (introContent){
-            return <BasicStaticSectionBody {..._.pick(introContent, 'content', 'filetype')} />;
-        }
-
-        return <p className="text-center">Introduction content not yet indexed.</p>;
-    }
-
     /**
      * The render function. Renders homepage contents.
      * @returns {Element} A React <div> element.
      */
     render() {
         const { windowWidth, context, session } = this.props;
-        const { announcements = null } = context || {};
+        const { announcements = null, content = [] } = context || {};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['announcements' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4527127)